Microsoft.Toolkit.Uwp.UI.Controls/AdaptiveGridView/AdaptiveGridView.Properties.cs

```diff
@@ -94,7 +88,8 @@ private static void OnOneRowModeEnabledChanged(DependencyObject d, object newVal
                     };
 
                     self._listView.SetBinding(GridView.MaxHeightProperty, b);
-                    self.VerticalScrollMode = ScrollMode.Disabled;
+                    ScrollViewer.SetVerticalScrollMode(self, ScrollMode.Disabled);
+                    ScrollViewer.SetVerticalScrollBarVisibility(self, ScrollBarVisibility.Hidden);
```

Comment:
This is the handler for when `OneRowModeEnabled` is changed. When this property is updated, how do you propose changing the scroll mode and visibility?